src/app/components/applications/SSV/MyAccount/components/Validator/SingleCluster/components/OperatorBox/OperatorBox.tsx

```diff
@@ -39,7 +40,7 @@ const OperatorBox = ({ operator }: { operator: any }) => {
             <Status item={operator}/>
             <Grid item className={classes.BoldText}>{isDeleted ? '-' : `${operator.performance['30d'].toFixed(2) ?? 0  }%`}</Grid>
             <Grid item
-                  className={classes.BoldText}>{isDeleted ? '-' : `${formatNumberToUi(walletStore.fromWei(operator.fee) * config.GLOBAL_VARIABLE.BLOCKS_PER_YEAR)  } SSV`}</Grid>
+                  className={classes.BoldText}>{isDeleted ? '-' : `${formatNumberToUi(ssvStore.newGetFeeForYear(walletStore.fromWei(operator.fee)))} SSV`}</Grid>
```

Comment:
Why the name is `newGetFeeForYear`? Is there some old get fee for year? 
If yes, there is no way for both of them to be used, let it be just `getFeeForYear`

Reply:
done